scripts/eosio_build_amazon.sh

```diff
@@ -56,11 +56,12 @@
 	if [[ "${OS_NAME}" == "Amazon Linux AMI" ]]; then
 		DEP_ARRAY=( git gcc72.x86_64 gcc72-c++.x86_64 autoconf automake libtool make bzip2 \
 		bzip2-devel.x86_64 openssl-devel.x86_64 gmp-devel.x86_64 libstdc++72.x86_64 \
-		python27.x86_64 python36-devel.x86_64 libedit-devel.x86_64 doxygen.x86_64 graphviz.x86_64)
+		python27.x86_64 python36-devel.x86_64 libedit-devel.x86_64 doxygen.x86_64 graphviz.x86_64
+      libusb1-devel libcurl-devel)
 	else
```

Comment:
tab vs spaces?

Reply:
good call thanks, we're usually so consistent I failed to notice it here